x/profiles/keeper/keeper_chain_link.go

```diff
@@ -0,0 +1,148 @@
+package keeper
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	sdkquery "github.com/cosmos/cosmos-sdk/types/query"
+
+	"github.com/desmos-labs/desmos/x/profiles/types"
+)
+
+// StoreChainLink stores the given chain link inside the current context.
+// It assumes that the given chain link has already been validated.
+func (k Keeper) StoreChainLink(ctx sdk.Context, link types.ChainLink, destinationAddress string) error {
```

Comment:
I think it would be better to: 
1. Rename the `destinationAddress` parameter to simply `user` 
2. Reorder the parameter making it `user string, link types.ChainLink`

This would make the method easier to comprehend what it does